### PR TITLE
Feature set for knotx-data-bridge

### PR DESCRIPTION
**Pull request by tMaxx**
This is the first working version of this module.

Feature set is limited, but compatible with all tests from knotx-data-bridge that were ported to JUnit 5 (see Knotx/knotx-data-bridge#19).

As per README, there are three main parts: KnotxExtension, KnotxWiremockExtension, and KnotxArgumentConverter (for parameterized tests).

Part of Cognifide/knotx#417.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,79 @@
-# kntox-junit5
-Testing Knot.x with JUnit 5
+# Knot.x JUnit 5
+JUnit 5 extensions and data type converters for Knot.x tests
+
+Provides following helpers:
+
+###KnotxExtension
```
Comment (Skejven) [inline README.md]: Please add space between `###` and title.
In that way markdown formatting will not apply (you may check it by using `View` button in the PR).

File: README.md [hunk 2]
```diff
@@ -1,2 +1,79 @@
...
+
+Knot.x-specific xtension that manages test Vert.x instances (with Knot.x configuration injection) and WireMock servers (through KnotxWiremockExtension).
```
Comment (Skejven) [inline README.md]: `* extension`

File: settings.gradle [hunk 3]
```diff
@@ -0,0 +1,10 @@
+/*
```
Comment (Skejven) [inline settings.gradle]: Let's change this header to the one with AL 2.0

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+tasks.withType(JavaCompile) {
+  options.encoding = 'UTF-8'
+  options.compilerArgs << '-Xlint:deprecation'
+  options.incremental = true
+}
+
+
+dependencies {
+  compile 'io.knotx:knotx-core'
```
Comment (tomaszmichalak) [inline build.gradle]: Why does knotx-junit5 depend on Knot.x Core. I think that the dependency should be: Knot.x Core depends on knotx-junit5.

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+  compile 'io.knotx:knotx-mocks'
```
Comment (tomaszmichalak) [inline build.gradle]: This dependency should be removed at all.

Comment (tMaxx) [reply build.gradle]: Will be removed in next version due for Knot.x Core, already deleted locally.

File: build.gradle (hunk 4 above)
Comment (tMaxx) [reply build.gradle]: Due to a requirement to mock `io.knotx.dataobjects.Fragment` via Mockito in `KnotxArgumentConverter` class. I don't currently have a sane workaround for this.

**Pull request merged**